libfwupd/fwupd-enums.h

```diff
@@ -284,63 +276,63 @@ typedef enum {
 	 *
 	 * Since: 0.6.3
 	 */
-	FWUPD_DEVICE_FLAG_REQUIRE_AC = 1ull << 3,
+	FWUPD_DEVICE_FLAG_REQUIRE_AC = 1ull << 2,
```

Comment:
Alas we can't easily "move them down" as the LVFS also uses the raw numbers IIRC. I think just remove FWUPD_DEVICE_FLAG_ONLY_OFFLINE and leave a "hole". I think the same for FWUPD_INSTALL_FLAG_OFFLINE too.

Reply:
We also store the flags in the history database -- and I *really* don't want to handle that migration :) I'll fix up your patch, I hope that's okay.